packages/dashboard/src/store/actions/changeDashboardGrid/changeCellSize.ts

```diff
@@ -0,0 +1,21 @@
+import { changeGridProperty } from './updateGrid';
+import type { Action } from 'redux';
+import type { DashboardState } from '../../state';
+
+type ChangeDashboardCellSizeActionPayload = {
+  cellSize: number;
+};
+export interface ChangeDashboardCellSizeAction extends Action {
+  type: 'CHANGE_CELL_SIZE';
+  payload: ChangeDashboardCellSizeActionPayload;
+}
+
+export const onChangeDashboardCellSizeAction = (
+  payload: ChangeDashboardCellSizeActionPayload
+): ChangeDashboardCellSizeAction => ({
+  type: 'CHANGE_CELL_SIZE',
+  payload,
+});
+
+export const changeDashboardCellSize = (state: DashboardState, action: ChangeDashboardCellSizeAction): DashboardState =>
+  changeGridProperty(state, 'cellSize', Math.max(0, action.payload.cellSize));
```

Comment:
Could you add a comment for why the Math.max is here? I can assume, but it's not obvious to me.

Reply:
👍 yes I can do that.